pom.xml

```diff
@@ -209,6 +209,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
+            <version>1.77</version>
```

Comment:
can't we use the one from the bom?

Reply:
once https://github.com/jenkinsci/bom/commit/0de33628090ece61378f6e8d97711b93af9b997a is in a release of the BOM, yes.

Reply:
I can add a todo to remove the version once BOM is released

Reply:
it would be great @alecharp , thanks!

Reply:
done in https://github.com/jenkinsci/ec2-plugin/pull/615/commits/06887408f6d0d7cc77c76fd6c74565b19df940d3